### DISCUSSION TITLE
Unable to connect to RavenDB via PostgreSQL ODBC driver

### DISCUSSION THREAD

**Discussion by hiralm21** [ravendb/ravendb#16401] [Q&A]
Hi,

I am new to RavenDB and trying to connect to it by setting up a PostgreSQL ODBC driver. I keep getting the below error. Can someone please help me?

![image](https://user-images.githubusercontent.com/70702878/236364136-f1a0f594-4880-4564-8b52-18a22d97c8b0.png)

**Comment by krivulcik** [comment]
You'll need to use one of the RavenDB clients available here: https://ravendb.net/download

You can find further information here: https://ravendb.net/docs/article-page/5.4/csharp/start/getting-started#client

**Reply by hiralm21** [reply]
It still didnt work

**Comment by ayende** [comment]
Postgres support is experimental.
I don't believe we tested the ODBC driver, primarily the .NET one.

**Reply by hiralm21** [reply]
Oh okay. Thank you. Are there any timelines on when it will be availlable?

**Reply by garayx** [reply]
Hi,

I have opened an issue to check if it can be added, https://issues.hibernatingrhinos.com/issue/RavenDB-20458
There are no timelines for this for issue now. 

Currently you can use the `Npgsql` data provider.

**Reply by hiralm21** [reply]
@garayx - Thank you so much!

**Reply by hiralm21** [reply]
@garayx - I am not sure I understand about Npgsql data provider. We want to setup an ODBC connection so that our applications downstream can talk to RavenDB using that ODBC data source.

**Reply by garayx** [reply]
as said above, Postgres support is experimental and Npgsql data provider was tested, ODBC data source is not supported yet.

**Reply by ppekrol** [reply]
Hi @hiralm21 

We are now in a process of finalizing the 6.0 release, afterwards we want to look into that. I really hope that we can provide you with that support sooner than later.

**Reply by hiralm21** [reply]
@ppekrol  - Thank you so much. Please keep us updated.
